include/revng-c/Tools/clift-opt/Model.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Model/Binary.h"
+
+extern TupleTree<model::Binary> Model;
```

Comment:
I guess this can become a `static` global in the `.cpp` file?

Reply:
I got rid of this and added a pass specific option. This was very ugly anyway. This model was filled from the command line by `clift-opt` `main` and used by the import model type passes.